scripts/markdown-transformers.ts

```diff
@@ -1,13 +1,14 @@
-/* eslint-disable import/no-extraneous-dependencies, no-console, no-useless-escape */
-const path = require('path');
+/* eslint-disable no-console, no-useless-escape */
```

Comment:
I'm torn here, inline disable or an `override` in `.eslintrc`. Thoughts?

```javascript
    overrides: [
        {
            files: [
                'scripts/*.ts'
            ],
            rules: {
                'no-console': 'off',
                'no-useless-escape': 'off'
            }
        },
   ]
```

Reply:
I removed the inline. I am a big fan of overrides over inlines typically.